app/src/main/java/com/obsidian/plugins/task_notifier/os/FIleObserverService.kt

```diff
@@ -94,11 +94,13 @@ class FileObserverService : Service() {
 
 
   private fun createForeGroundNotification(context: Context) {
+    NotificationManager.ensureForegroundChannelExists(context);
     val notification = NotificationManager.notify(
       context,
       "Obisidian Reminder running",
       "background",
-      99
+      99,
+      NotificationManager.NOTIFICATION_FOREGROUND_CHANNEL_ID
     )
     startForeground(1, notification)
```

Comment:
I think we should think about this hardcoded 1. What happens if we are watching multiple obsidian vaultts? Maybe using guid for this would be sufficient?

Reply:
We will have 1 foreground service so it can be work with multiple vaults.
ID has been changed from 1 to 99 because the file changed event and this background event used the same. But now with separated channels we can change it. But what do you think about the file changed event? Is it necessary? In my opinion we should move it to foreground channel and keep clean the reminder list

Reply:
Guid based reqId can work everywhere

Reply:
well. i completly misundestood the nature of the Service in android. I thought this codes starts a new service instance for each path:
![image](https://user-images.githubusercontent.com/53854777/219938939-0ce5b476-5612-47bf-97bb-e7d0b65f8fc5.png)

However it is a singleton which means inside the FileObserverService we have to manage the list of the paths which have to be watched and keep the FileObserver instances for them alive in the FileObserverService